mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+# Getting Started
+
+Welcome to the "Getting Started" guide. This chapter will lead you through the initial steps of logging into the HPC-system and submitting your very first job. We'll also walk you through the process step by step using a practical example.
+
+In addition to this chapter, you might find these  [slides](https://www.ugent.be/hpc/en/training/2023/introhpc202306) to be useful resources.
+
+Before proceeding, read [the introduction to HPC](introduction.md) to gain an understanding of the HPC infrastructure and its terminology.
+
+### Getting Access
+
+To get access, visit <https://docs.hpc.ugent.be/account>.
+
+If you have not used linux before, now would be a good time to follow our [Linux Tutorial]().
+  
+#### A typical workflow looks like this:
+
+1.  Connect to the login nodes 
+3.  Transfer your files to the cluster 
+3.  Optional: compile your code and test it 
+4.  Create a job script and submit your job
+5.  Wait for job to be executed
+6.  Study the results generated by your jobs, either on the cluster or
+    after downloading them locally.
+
+We will walk through an illustrative workload to get you started. In this example, our objective is to train a deep learning model for recognizing hand-written digits (MNIST dataset).
+See the [example](https://github.com/EESSI/eessi-demo/tree/main/TensorFlow).
```

Comment:
We should copy the example in a subdirectory of this repo, and point to that location instead, since the script in `EESSI/eessi-demo` may change in ways the cause trouble in these docs here.